clients/fossid-webapp/src/test/kotlin/FossId2023dot1Test.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.types.shouldBeInstanceOf
+
+import org.ossreviewtoolkit.clients.fossid.FossIdRestService
+import org.ossreviewtoolkit.clients.fossid.FossIdServiceWithVersion
+import org.ossreviewtoolkit.clients.fossid.VersionedFossIdService2021dot2
+import org.ossreviewtoolkit.clients.fossid.checkResponse
+import org.ossreviewtoolkit.clients.fossid.deleteScan
+
+private const val PROJECT_CODE = "semver4j"
+private const val SCAN_CODE_2021_2 = "${PROJECT_CODE}_20201203_090342"
+
+/**
+ * This client test tests the calls that have been changed for FossID 2023.1.
+ * This version of FossID changed the return type of the deleteScan function: it was before an integer, and it is now an
+ * map of strings to strings. Creating a special [FossIdServiceWithVersion] implementation for this call is an overkill
+ * as ORT does not even use the return value. Therefore, this change is handled by the Jackson deserializer.
```

Comment:
At least this part of the docs to me sound like something that should go to the deserializer, not to this test.